content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/file-and-directory-naming/general
+ready: true
+tags:
+- File and directory naming
+- python
+title: File and directory naming in Python
+---
+
+## How should I name my PYTHON modules and packages?
+
+
+Modules should have short, all-lowercase names and underscores can be used to improves readability. 
+
+Packages should also have short, all-lowercase names, although the use of underscores is discouraged.
+
+
+#### NO:
+
+    TASK/
+        1task.py
+        task 1.py
+        task1.1.py
+        
+    passwordChecker/
+        add-logger.py
+
+    snakecase/
+        productionnotes.py
+        keepItShortAndmeaningfull.py
+
+#### Yes:
+    task/
```

Comment:
maybe saying things in an explicit way would be useful as well.

Eg: your file and directory names should follow the same kinds of naming conventions as python variables. That means that they should be small letters and words should be seperated by underscores. Python files always have a .py extension